vllm/spec_decode/spec_decode_worker.py

```diff
@@ -632,10 +661,12 @@ def _verify_tokens(
             accepted_index = accepted_token_ids + 1  # Convert -1 to 0
             accepted_index = accepted_index.count_nonzero(dim=1).add_(-1)
             index = accepted_index[:, None, None].expand(-1, 1, hs_size)
+            bonus_token_previous_hidden_states = hidden_states[:, -2]  # b x d
```

Comment:
nit : Wondering if we can rename bonus_token_previous_hidden_states to last_non_bonus_token_hidden_states?